README.md

```diff
@@ -171,7 +171,9 @@ NOTES:
 |HLS       |**Y**     |**Y**    |**Y** ¹  | -        |
 
 NOTES:
- - ¹: We support FairPlay through Apple's native HLS player.
+ - ¹: We support FairPlay through MSE/EME when available and when
+   `useNativeHlsOnSafari` is turned off. Otherwise, we are using
```

Comment:
Although, on second thought, does it even make sense for `useNativeHlsOnSafari` to be set to `true` by default, with this PR? If we change it to be `false` by default, we won't even need to have a footnote here.

Reply:
Given the limitations of MSE with FPS (eg TS is not supported), I think it is necessary to keep it to true.

Reply:
And there's still the issue with the multi-keys CMAF HLS, for which I will add a note in the documentation